content/java/concepts/array-list/terms/subList/subList.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.subList()'
+Description: 'Learn about the .subList() method in Java ArrayList, allowing you to obtain a portion of the list based on specified indices.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Method'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+## Introduction
+
+In Java, `.subList()` method is a powerful feature of the [`ArrayList`](https://www.codecademy.com/resources/docs/java/array-list) class that allows you to obtain a portion of the list based on specified indices. This method provides a convenient way to create a sublist containing elements from the original `ArrayList`.
+
+## Syntax
+
+```pseudo
+List<E> subList (int fromIndex, int toIndex)
+```
+
+The `.subList()` method is invoked on an ArrayList object and taks two parameters: `fromIndex` (inclusive) and `toIndex` (exclusive). It returns a view of the portion of the original list between the specified indices.
```

Comment:
Remove this line and add the description of `fromIndex ` and `toIndex` parameters below the syntax block.